src/api/helpers/ApiRequestTypes.ts

```diff
@@ -0,0 +1,24 @@
+export interface RequestConfigType {
```

Comment:
We have common convention about naming types and interfaces. If we use interface, better if we name like this: `export interface IRequestConfig`. Also, we can use the 'Type' word with type.

Reply:
Fix this